src/Zenith.sol

```diff
@@ -82,6 +83,22 @@ contract Zenith is Passage {
         sequencerAdmin = _sequencerAdmin;
     }
 
+    /// @notice Returns the next sequence number.
+    /// @dev Because sequences must start at 1, we add 1 to the mapping value.
+    /// @param _rollupChainId - the chainId of the rollup chain. Any chainId is accepted by the contract.
+    /// @return The next sequence number.
+    function nextSequence(uint256 _rollupChainId) public view returns (uint256) {
+        return sequences[_rollupChainId] + 1;
+    }
+
+    /// @notice Increments the sequence number and returns it.
+    /// @dev Because sequences must start at 1, we add 1 to the mapping value.
+    /// @param _rollupChainId - the chainId of the rollup chain. Any chainId is accepted by the contract.
+    /// @return The next sequence number.
+    function incrementSequence(uint256 _rollupChainId) internal returns (uint256) {
+        return ++sequences[_rollupChainId];
```

Comment:
not sure this needs to be its own function? i guess it's nice for having clear docs for the behavior, but feels a little heavy-handed